### ISSUE TITLE
Help needed to run SparkBWA without Hadoop

### ISSUE THREAD

**Issue by avapirev** [citiususc/SparkBWA#36]
Hi, Is there any intention to enable SparkBWA to run with only Spark's native RDD and native scheduler without going through the hassle of HDFS. The latter needs a Hadoop running and that could be a huge hiccup if one wants to run it under Torque for example. In contrast, running the Spark local scheduler under PBS/Torque is an easy and smooth process since it does not require to have Hadoop running on each compute node in a HPC cluster setup.

Basically, if someone could give me few hints how to save the output to a file without first transferring the RDDs to HDFS and writing the final SAM there.

I am not a Java person (read zero knowledge) but what I can gather the 4 files to be modified are Bwa.java, BwaOptions.java, BwaAlignementBase.java (after around lines 135), BwaInterpreter.java.

Greatly appreciated.

**Comment by xubo245** [comment]
SparkBWA input/output file from/to HDFS.

You can replace it with you code.

**Comment by avapirev** [comment]
I would've stripped down the HDFS I/O from the code myself if I knew java. I guess I will be reading about it these days.

**Comment by avapirev** [comment]
Resolved by @xubo245  in Issue #22
